daemon/graphdriver/aufs/aufs.go

```diff
@@ -272,16 +290,44 @@ func (a *Driver) Remove(id string) error {
 	if !exists {
 		mountpoint = a.getMountpoint(id)
 	}
-	if err := a.unmount(mountpoint); err != nil {
-		// no need to return here, we can still try to remove since the `Rename` will fail below if still mounted
-		logrus.Debugf("aufs: error while unmounting %s: %v", mountpoint, err)
+
+	var retries int
+	for {
+		mounted, err := a.mounted(mountpoint)
+		if err != nil {
+			return err
+		}
+		if !mounted {
+			break
+		}
+
+		if err := a.unmount(mountpoint); err != nil {
```

Comment:
Most of the aufs EBUSY issues point to error on removal. Since we are debugging this specific issue, I wanted to keep it to this scope, rather than move it to the generic unmount method.